<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save / Load animation</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbShortNames">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="toolTip">
         <string>will look for a node only by short name.

Unpredictable results if multiple nodes with same names in different hierarchies exist.</string>
        </property>
        <property name="text">
         <string>use short names (if hierarchy has changed)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbReplaceKeys">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777213</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete animation curves of the affected node 
before applying new animation</string>
          </property>
          <property name="text">
           <string>REPLACE animation curve(s)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rbMergeKeys">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Pastes in keys without removing existing animations on node</string>
          </property>
          <property name="text">
           <string>MERGE with existing animation curve(s)</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolTip">
           <string>add animation starting on frame x</string>
          </property>
          <property name="text">
           <string>Frame offset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="fOffset">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>18</height>
           </size>
          </property>
          <property name="toolTip">
           <string>add animation starting on frame x</string>
          </property>
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbNameSpaceFromAnim">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>This function will use the names from animation file 
without changing namespaces</string>
          </property>
          <property name="text">
           <string>use namespace(s) from animation file</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rbNameSpaceOverride">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>This option will remove namespaces if empty 
or overwrite namespaces from animation file with new namespace</string>
          </property>
          <property name="text">
           <string>override (empty means CUT Namespace)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="tbNamespace">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnNamespace">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>gets the namespace from currently selected scene node</string>
          </property>
          <property name="text">
           <string>get namespace(s) from selection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>12</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbForceNameSpace">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>10</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;use this if you have namespaces on nodes but animation was saved:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;- with &amp;quot;cut namespaces&amp;quot; option enabled or&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;- from nodes without namespaces&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>assume namespace(s) on complete hierarchy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>LOAD ANIMATION</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbSelection">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Save animation of selected scene nodes</string>
          </property>
          <property name="text">
           <string>Selected nodes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rbMethod1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Will look for controllers of 1 or more characters 
with the help of rig node network with custom nodes. 
(needs a selection if more characters exist in scene)</string>
          </property>
          <property name="text">
           <string>CONTROLLERS from RIGNODE NETWORK</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string>Options</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="cbNamespace">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>cutting namespaces from nodes</string>
          </property>
          <property name="text">
           <string>cut namespace(s)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="cbMultiChar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>use this option if you want to export animations 
from ALL characters in the current scene into a single animation file 
- will not allow you to cut namespaces</string>
          </property>
          <property name="text">
           <string>export multiple characters</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbMethod0">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Save animation from all controls of 1 or more characters 
by the selection set method 
(looking for a selection set with an attribute called CryCON or ccon) 
- a node has to be selected if more characters exist in scene</string>
          </property>
          <property name="text">
           <string>CONTROLLERS from SET</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lUserName">
          <property name="text">
           <string>UserName</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tbUserName">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="tbRigVersion"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lRigVersion">
          <property name="text">
           <string>RigVersion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAVE ANIMATION</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbMethod0</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbMultiChar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMethod1</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbMultiChar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMultiChar</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbNamespace</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNameSpaceFromAnim</sender>
   <signal>toggled(bool)</signal>
   <receiver>tbNamespace</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbReplaceKeys</sender>
   <signal>clicked(bool)</signal>
   <receiver>rbMergeKeys</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMergeKeys</sender>
   <signal>clicked(bool)</signal>
   <receiver>rbReplaceKeys</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNameSpaceFromAnim</sender>
   <signal>clicked(bool)</signal>
   <receiver>rbNameSpaceOverride</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNameSpaceOverride</sender>
   <signal>clicked(bool)</signal>
   <receiver>rbNameSpaceFromAnim</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNameSpaceFromAnim</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbForceNameSpace</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
